website/snippets/_v2-sl-prerequisites.md

```diff
@@ -0,0 +1,55 @@
+
+<VersionBlock firstVersion="1.6">
+
+- Have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America
+- Have both your production and development environments running dbt version 1.6 or higher 
+- Use Snowflake, BigQuery, Databricks, and Redshift data platform
+- A successful run in the environment where your Semantic Layer is configured
+  * Note &mdash; Deployment environment is currently supported (_development experience coming soon_)
+- Install the [MetricFlow CLI](/docs/build/metricflow-cli). After installing the package, make sure you run at least one model.
+- Set up the [Semantic Layer API](/docs/dbt-cloud-apis/sl-api-overview) in the integrated tool to import metric definitions 
+  * Developer accounts will be able to query manually using the [MetricFlow CLI](/docs/build/metricflow-cli) and SQL. To dynamically query metrics using external tools, you'll need to have a dbt Cloud Team or Enterprise account with access to the Semantic Layer API.<br />
+- Have an understanding of [MetricFlow's](/docs/build/about-metricflow) key concepts, which powers the revamped dbt Semantic Layer.
+
+
+</VersionBlock>
+
+
+<VersionBlock lastVersion="1.5">
+
+- Have a multi-tenant dbt Cloud instance, <a href="https://docs.getdbt.com/docs/cloud/about-cloud/regions-ip-addresses">hosted</a> in North America <br />
+- Have both your production and development environments running dbt version 1.3 or higher <br />
+- Use Snowflake data platform <br />
+- Install the <a href="https://hub.getdbt.com/dbt-labs/metrics/latest/">dbt metrics package</a> version <code>>=1.3.0, <1.4.0</code> in your dbt project <br />
+  * **Note** &mdash; After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
+- Set up the <a href="https://docs.getdbt.com/docs/dbt-cloud-apis/discovery-api">Discovery API</a> in the integrated tool to import metric definitions 
+  * Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Discovery API <br />
+- Recommended - Review the <a href="https://docs.getdbt.com/docs/build/metrics">dbt metrics page</a> and <a href="https://docs.getdbt.com/blog/understanding-the-components-of-the-dbt-semantic-layer">Understanding the components of the dbt Semantic Layer</a> blog <br />
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.3" lastVersion="1.3" >
```

Comment:
There's not too much different between 1.2 and 1.3, so can we maybe only version the two lines affected instead of the whole block? It might make the code easier to read for the next person. Alternately, you could create a global variable and [version that](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-global-variables).